### ISSUE TITLE
Is there a way to create additional gvisor configurations?

### ISSUE THREAD

**Issue by ruifung** [siderolabs/extensions#166]
Or is the only way to make a separate system extension / modify the existing one?

Just asking in case I need to change the gvisor configuration for example, enabling root fs overlay, or changing the platform used in gvisor.

**Comment by smira** [comment]
you can of course build your own version of the system extension. if there are valuable config options, let's have them discussed here and how to make that configuration available.

**Comment by SISheogorath** [comment]
Yes, I'm really looking into using kvm as platform for gvisor. Wouldn't it be a simple solution, to move the config to `/var` to allow machine files to take care of it?
